src/Http/Http.Abstractions/src/StatusCodes.cs

```diff
@@ -4,10 +4,13 @@
 namespace Microsoft.AspNetCore.Http;
 
 /// <summary>
-/// A collection of constants for HTTP status codes.
-///
-/// Status Codes listed at http://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml
+/// A collection of constants for
+/// <see href="http://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml" >HTTP status codes</see >.
```

Comment:
I'm hesitant to use `<see href=...` in a bunch of places since it breaks the summary in intellisense. See https://github.com/dotnet/aspnetcore/pull/38659#discussion_r760672551

Reply:
I thought it is broken when the link is relative?

Reply:
I wasn't able to reproduce this issue, the href worked fine for me locally in VS 2022.